pkg-manager/get-context/src/readLockfiles.ts

```diff
@@ -128,6 +129,7 @@ export async function readLockfiles (
     existsCurrentLockfile: files[1] != null,
     existsWantedLockfile: files[0] != null && !isEmptyLockfile(wantedLockfile),
     wantedLockfile,
+    wantedLockfileUnmodified: JSON.parse(JSON.stringify(wantedLockfile)),
```

Comment:
nit: It might be worth a comment that we have to recreate this object since the consumer is expected to mutate `wantedLockfile`. But maybe that's fairly obvious and isn't worth commenting.

Reply:
I was also considering just to save the hash of the object and then comparing hashes. That would use less memory but probably would do more processing

Reply:
I have changed it. There was actually no need in the copy.